eml/screens/excercise/ExerciseScreen.jsx

```diff
@@ -91,8 +91,8 @@ export default function ExerciseScreen({ givenId = '65181a4f4c78b45368126ed7', o
   useEffect(() => {
     const fetchData = async () => {
       try {
-        setExerciseData(exercise = await getExerciseByid(givenId));
-        setSectionData(section = await getSectionByid(exercise.parentSection));
+        setExerciseData(await getExerciseById(givenId));
+        setSectionData(await getSectionByid(exercise.parentSection));
```

Comment:
Probably also change the naming of getSectionByid to getSectionById

Reply:
This have been done now